app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::CommentsController < ApiController
+  before_action :authorize_request, only: [:create]
+
+  def index
+    # /api/v1/users/:user_id/posts/:post_id/comments(.:format)
```

Comment:
You did a great job!

- [ ] You have one more step to get approved. To get approved, please remove unused commented code from this file :)

Reply:
Thanks, @Rustamxon7 . We completely forgot to remove it!. 🙏